src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/config/HmppsRestrictedPatientsApiExceptionHandler.kt

```diff
@@ -173,7 +173,6 @@ class HmppsRestrictedPatientsApiExceptionHandler {
 
   @ExceptionHandler(EntityNotFoundException::class)
   fun handleEntityNotFoundException(e: Exception): ResponseEntity<ErrorResponse> {
-    log.debug("Not found (404) returned with message {}", e.message)
```

Comment:
this is unnecessary since app insights logs it as in `requests` anyway.  Furthermore it occurs around 7,000 times an hour since it is called by DPS.